libcontainerd/local/local_windows.go

```diff
@@ -941,7 +942,7 @@ func (t *task) Summary(_ context.Context) ([]libcontainerdtypes.Summary, error)
 	for i := range p {
 		pl[i] = libcontainerdtypes.Summary{
 			ImageName:                    p[i].ImageName,
-			CreatedAt:                    p[i].CreateTimestamp,
+			CreatedAt:                    timestamppb.New(p[i].CreateTimestamp),
```

Comment:
Oh; didn't make this change in the other PR; was this one related to the hcsshim update or to the grpc/proto updates?

Reply:
Seems related to hcsshim
```
# github.com/docker/docker/libcontainerd/local
..\..\libcontainerd\local\local_windows.go:944:34: cannot use p[i].CreateTimestamp (variable of type time.Time) as *timestamppb.Timestamp value in struct literal
```